sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/entries/CreateNewTravelEntrySteps.java

```diff
@@ -239,7 +239,7 @@ public CreateNewTravelEntrySteps(
               "Communities are not equal");
           softly.assertEquals(
               aCase.getPointOfEntry(),
-              travelEntry.getPointOfEntry() + " (Inaktiv)",
+              travelEntry.getPointOfEntry(),
```

Comment:
Won't this fail when the point of entry **does** show up as inactive?
Alwin said that "`Within this test both should be acceptable`" so I think we need the assert to alternate between active and inactive version instead of expecting only one of them.

Reply:
"The presents of "inactive" is related to the status (active/archived) of the related destrict record. Within this test both should be acceptable. We will test the "inactive" feature separately in [SORQA-199]" - in this test case it will always 'aktiv' which is also correct.